Sources/SwiftyMocky/MockyAssert.swift

```diff
@@ -45,12 +51,33 @@ public func MockyAssert(
 private func XCTMockyAssert(
     _ expression: @autoclosure () -> Bool,
     _ message: @autoclosure () -> String = "Verify failed",
+    fileId: StaticString = #fileID,
+    filePath: StaticString = #filePath,
     file: StaticString = #file,
-    line: UInt = #line
+    line: UInt = #line,
+    column: UInt = #column
 ) {
-    #if canImport(XCTest)
-    XCTAssert(expression(), message(), file: file, line: line)
-    #else
+#if canImport(Testing)
+    if Test.current != nil {
+        if !expression() {
+            Issue.record(
+                Comment(rawValue: message()),
+                sourceLocation: .init(
+                    fileID: fileId.description,
+                    filePath: filePath.description,
+                    line: Int(line),
+                    column: Int(column)
+                )
+            )
+        }
+        // Return so we do not call XCTAssert if we are in a Swift Testing test.
+        return
+    }
+#endif
+    
+#if canImport(XCTest)
+    XCTAssert(expression(), message(), file: filePath, line: line)
```

Comment:
I changed file to filePath here because XCTAssert's default parameter is #filePath as well. Is this a correct thing to do? It still shows up at the correct place in source in the editor.